debugging/Technical_Overview.md

```diff
@@ -67,7 +67,7 @@ There are 9 compatible 10-pin hardware modules for Tessel 2 available at the tim
 * [Servo Module](https://github.com/tessel/servo-pca9685)
 
 ##### USB Modules
-USB Modules haven't been released yet so some don't have Github repos and many are in progress. In general, Tessel supports the following kind of USB modules which you can find from many online vendors:
+Tessel supports the following kind of USB modules which you can find from many online vendors:
```

Comment:
Maybe "Tessel supports interfacing with USB devices. Here are libraries we support for Tessel to interoperate with USB modules, available from many vendors:"